
2025-06-28 18:16:32 [INFO] PUT FormData processed
2025-06-28 18:16:32 [WARN] No new file uploaded for update, keeping existing:
2025-06-28 18:16:32 [INFO] Book updated: nature 
2025-06-30 10:58:30 [INFO] Book issue updated successfully: 4
2025-06-30 12:14:31 [INFO] PUT FormData processed
2025-06-30 12:14:31 [WARN] No new file uploaded for update, keeping existing:
2025-06-30 12:14:31 [INFO] Book updated: nature 
2025-06-30 13:58:39 [INFO] Book returned successfully: 4
2025-06-30 14:02:08 [INFO] Book issue updated successfully: 4
2025-06-30 14:02:15 [INFO] Book renewed successfully: 4 for 7 days
2025-06-30 14:02:28 [INFO] Book issue updated successfully: 4
2025-06-30 14:24:17 [INFO] Book issue updated successfully: 4
2025-06-30 14:25:33 [INFO] Book issue updated successfully: 4
2025-06-30 14:29:43 [INFO] Book returned successfully: 4
2025-06-30 15:10:56 [INFO] Book issue updated successfully: 4
2025-06-30 16:03:31 [INFO] Book returned successfully: 4
2025-06-30 16:36:17 [INFO] Book returned successfully: 4
2025-06-30 16:41:48 [INFO] Book issued successfully: 1 to student 65
2025-06-30 16:45:17 [INFO] Book returned successfully: 5
2025-06-30 16:48:01 [INFO] Book returned successfully: 4
2025-06-30 16:55:24 [INFO] Book returned successfully: 4
2025-06-30 17:01:37 [INFO] Book issue updated successfully: 5
2025-06-30 17:08:05 [ERROR] Transaction failed during book issue deletion
RequestError: The DELETE statement conflicted with the REFERENCE constraint "FK__Penalty__IssueId__16644E42". The conflict occurred in database "jkconsultancyadmindb", table "dbo.Penalty", column 'IssueId'.
    at handleError (C:\Users\Windows\Desktop\New Projetct\library-management-system\.next\server\chunks\node_modules_mssql_93e55d20._.js:3584:23)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\Users\Windows\Desktop\New Projetct\library-management-system\.next\server\chunks\node_modules_tedious_lib_connection_cdf92d37.js:879:22)
    at RequestTokenHandler.onErrorMessage (C:\Users\Windows\Desktop\New Projetct\library-management-system\.next\server\chunks\node_modules_tedious_lib_token_778d00c4._.js:2122:25)
    at Readable.<anonymous> (C:\Users\Windows\Desktop\New Projetct\library-management-system\.next\server\chunks\node_modules_tedious_lib_token_778d00c4._.js:1823:39)
    at Readable.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
    at Readable.push (node:internal/streams/readable:393:5)
    at nextAsync (node:internal/streams/from:194:22)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-06-30 17:08:05 [ERROR] Error deleting book issue
RequestError: The DELETE statement conflicted with the REFERENCE constraint "FK__Penalty__IssueId__16644E42". The conflict occurred in database "jkconsultancyadmindb", table "dbo.Penalty", column 'IssueId'.
    at handleError (C:\Users\Windows\Desktop\New Projetct\library-management-system\.next\server\chunks\node_modules_mssql_93e55d20._.js:3584:23)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\Users\Windows\Desktop\New Projetct\library-management-system\.next\server\chunks\node_modules_tedious_lib_connection_cdf92d37.js:879:22)
    at RequestTokenHandler.onErrorMessage (C:\Users\Windows\Desktop\New Projetct\library-management-system\.next\server\chunks\node_modules_tedious_lib_token_778d00c4._.js:2122:25)
    at Readable.<anonymous> (C:\Users\Windows\Desktop\New Projetct\library-management-system\.next\server\chunks\node_modules_tedious_lib_token_778d00c4._.js:1823:39)
    at Readable.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
    at Readable.push (node:internal/streams/readable:393:5)
    at nextAsync (node:internal/streams/from:194:22)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-06-30 17:08:18 [ERROR] Transaction failed during book issue deletion
RequestError: The DELETE statement conflicted with the REFERENCE constraint "FK__Penalty__IssueId__16644E42". The conflict occurred in database "jkconsultancyadmindb", table "dbo.Penalty", column 'IssueId'.
    at handleError (C:\Users\Windows\Desktop\New Projetct\library-management-system\.next\server\chunks\node_modules_mssql_93e55d20._.js:3584:23)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\Users\Windows\Desktop\New Projetct\library-management-system\.next\server\chunks\node_modules_tedious_lib_connection_cdf92d37.js:879:22)
    at RequestTokenHandler.onErrorMessage (C:\Users\Windows\Desktop\New Projetct\library-management-system\.next\server\chunks\node_modules_tedious_lib_token_778d00c4._.js:2122:25)
    at Readable.<anonymous> (C:\Users\Windows\Desktop\New Projetct\library-management-system\.next\server\chunks\node_modules_tedious_lib_token_778d00c4._.js:1823:39)
    at Readable.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
    at Readable.push (node:internal/streams/readable:393:5)
    at nextAsync (node:internal/streams/from:194:22)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-06-30 17:08:18 [ERROR] Error deleting book issue
RequestError: The DELETE statement conflicted with the REFERENCE constraint "FK__Penalty__IssueId__16644E42". The conflict occurred in database "jkconsultancyadmindb", table "dbo.Penalty", column 'IssueId'.
    at handleError (C:\Users\Windows\Desktop\New Projetct\library-management-system\.next\server\chunks\node_modules_mssql_93e55d20._.js:3584:23)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\Users\Windows\Desktop\New Projetct\library-management-system\.next\server\chunks\node_modules_tedious_lib_connection_cdf92d37.js:879:22)
    at RequestTokenHandler.onErrorMessage (C:\Users\Windows\Desktop\New Projetct\library-management-system\.next\server\chunks\node_modules_tedious_lib_token_778d00c4._.js:2122:25)
    at Readable.<anonymous> (C:\Users\Windows\Desktop\New Projetct\library-management-system\.next\server\chunks\node_modules_tedious_lib_token_778d00c4._.js:1823:39)
    at Readable.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
    at Readable.push (node:internal/streams/readable:393:5)
    at nextAsync (node:internal/streams/from:194:22)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-06-30 17:21:20 [INFO] Book returned successfully: 4
2025-06-30 17:21:54 [ERROR] Transaction failed during book issue deletion
RequestError: The DELETE statement conflicted with the REFERENCE constraint "FK__Penalty__IssueId__16644E42". The conflict occurred in database "jkconsultancyadmindb", table "dbo.Penalty", column 'IssueId'.
    at handleError (C:\Users\Windows\Desktop\New Projetct\library-management-system\.next\server\chunks\node_modules_mssql_93e55d20._.js:3584:23)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\Users\Windows\Desktop\New Projetct\library-management-system\.next\server\chunks\node_modules_tedious_lib_connection_cdf92d37.js:879:22)
    at RequestTokenHandler.onErrorMessage (C:\Users\Windows\Desktop\New Projetct\library-management-system\.next\server\chunks\node_modules_tedious_lib_token_778d00c4._.js:2122:25)
    at Readable.<anonymous> (C:\Users\Windows\Desktop\New Projetct\library-management-system\.next\server\chunks\node_modules_tedious_lib_token_778d00c4._.js:1823:39)
    at Readable.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
    at Readable.push (node:internal/streams/readable:393:5)
    at nextAsync (node:internal/streams/from:194:22)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-06-30 17:21:54 [ERROR] Error deleting book issue
RequestError: The DELETE statement conflicted with the REFERENCE constraint "FK__Penalty__IssueId__16644E42". The conflict occurred in database "jkconsultancyadmindb", table "dbo.Penalty", column 'IssueId'.
    at handleError (C:\Users\Windows\Desktop\New Projetct\library-management-system\.next\server\chunks\node_modules_mssql_93e55d20._.js:3584:23)
    at Connection.emit (node:events:518:28)
    at Connection.emit (C:\Users\Windows\Desktop\New Projetct\library-management-system\.next\server\chunks\node_modules_tedious_lib_connection_cdf92d37.js:879:22)
    at RequestTokenHandler.onErrorMessage (C:\Users\Windows\Desktop\New Projetct\library-management-system\.next\server\chunks\node_modules_tedious_lib_token_778d00c4._.js:2122:25)
    at Readable.<anonymous> (C:\Users\Windows\Desktop\New Projetct\library-management-system\.next\server\chunks\node_modules_tedious_lib_token_778d00c4._.js:1823:39)
    at Readable.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushObjectMode (node:internal/streams/readable:538:3)
    at Readable.push (node:internal/streams/readable:393:5)
    at nextAsync (node:internal/streams/from:194:22)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2025-06-30 17:23:42 [INFO] Book issue deleted successfully: 5
2025-06-30 17:27:00 [INFO] Book issue updated successfully: 4
2025-06-30 17:27:15 [INFO] Book issue updated successfully: 4
2025-06-30 17:27:44 [INFO] Book returned successfully: 4
2025-06-30 17:30:27 [INFO] Book returned successfully: 4
